core-services/prow/02_config/_plugins.yaml

```diff
@@ -185,7 +185,12 @@ approve:
   repos:
   - openshift-priv/openshift-tests
   - openshift/openshift-tests
-  require_self_approval: false
+  require_self_approval: true
+- commandHelpLink: https://go.k8s.io/bot-commands
+  repos:
+  - openshift-priv/openshift-tests-private
+  - openshift/openshift-tests-private
```

Comment:
This is basically the same settings lieke the `openshift-tests` repos above, so maybe just add these repos to the list of repos above?

Reply:
Yes, thanks!